src/Symfony/Component/Yaml/Escaper.php

```diff
@@ -48,7 +52,10 @@ class Escaper
      */
     public static function requiresDoubleQuoting($value)
     {
-        return preg_match('/'.self::REGEX_CHARACTER_TO_ESCAPE.'/u', $value);
+        // Removes quotes and slashes
+        $mask = implode('', array_slice(self::$escapees, 5));
```

Comment:
we'd better create a new const for this, and deprecate REGEX_CHARACTER_TO_ESCAPE meanwhile